src/routes/tasks/components/TaskOutputsDrawerSection.tsx

```diff
@@ -130,7 +130,7 @@ const TaskOutputsDrawerSection = ({
                             ))}
                         {!taskLoading &&
                             task &&
-                            Object.values(taskOutputsAssets).map((output) => {
+                            taskOutputsAssets.map((output) => {
```

Comment:
Why is the Object.values not used anymore?

Reply:
Because taskOutputsAssets is already an array